Cloud drives M35 documentation (#251)
exoplatform/exo-documentation: docs/GettingStarted.rst

.. note:: The supported video players are Youtube, Vimeo and Dailymotion.

.. _CloudDrives:

Connecting to drives
~~~~~~~~~~~~~~~~~~~~~

> It is possible to attach documents from one of your external drives by connecting to them from the composer's "Select from drives" drawer.

Reviewer: ```suggestion
With the V6 of eXo Platfom, you are able to attach documents from your external drives. For that, you need first to connect to your external drive through the "Select from drives" drawer.
```

|image247|

> From the root folder "Drives", the folders are organized by blocks :

Reviewer: ```suggestion
From the root folder "Drives", folders are organized by blocks :
```

My drives: Contains your personal documents and shortcuts to your connected cloud drives.

My spaces : Contains folders from the spaces in which you are a member.

> Others : displays "Users" folder if you are a regular member and other folders relatef to admins if you have the admin role.

Reviewer: ```suggestion
Others : displays "Users" folder if you are a regular member and other administrative folders if you belong to administrators group.
```

To connect to a cloud drive account, you must use use the icon |image248|.

> This icon redirects you to "Connect your service" drawer that allows you to choose either to connect to a google drive account or a one drive account.

Reviewer: ```suggestion
It redirects you to "Connect your service" drawer allowing to choose either to connect to a google drive account or a one drive account.
```
